### PR TITLE
Feat: Proxy health supports selecting clusters via -c or --cluster

### PR DESCRIPTION
**Pull request by yue9944882**

### CODE REVIEW

File: pkg/cmd/proxy/health/exec.go [hunk 1]
```diff
@@ -150,8 +150,10 @@ func (o *Options) run(streams genericclioptions.IOStreams) error {
 
 	w := newWriter(streams)
 	for _, cluster := range managedClusterList.Items {
-		if err := o.visit(&w, hubRestConfig, addonClient, tunnel.DialContext, cluster.Name); err != nil {
-			klog.Errorf("An error occurred when requesting: %v", err)
+		if len(o.clusters) == 0 || contains(o.clusters, cluster.Name) {
```
Comment (qiujian16) [inline pkg/cmd/proxy/health/exec.go:153]: how about putting o.clusters to a sets.String?

Comment (yue9944882) [reply pkg/cmd/proxy/health/exec.go]: updated

File: pkg/cmd/proxy/health/exec.go [hunk 2]
```diff
@@ -329,3 +333,12 @@ func (w *writer) print(clusterName, installed, available, health, latency string
 func (w *writer) flush() {
 	_ = w.w.Flush()
 }
+
+func contains(vs []string, target string) bool {
```
Comment (qiujian16) [inline pkg/cmd/proxy/health/exec.go:337]: this func is not needed, right?

Comment (yue9944882) [reply pkg/cmd/proxy/health/exec.go]: nice catch

**Pull request merged**